xDrip/Controllers/CGMController.swift

```diff
@@ -14,7 +14,7 @@ final class CGMController {
     private init() {}
     
     // MARK: Connection
-    typealias ConnectionCallback = (Bool) -> Void
+    typealias ConnectionCallback = (Bool, Bool) -> Void
```

Comment:
If we have multiple tuple arguments (especially with the same type), they should be either named or replaced with a struct.